common/server_protocolhandler.cpp

```diff
@@ -373,6 +390,22 @@ void Server_ProtocolHandler::pingClockTimeout()
 
     if (timeRunning - lastDataReceived > server->getMaxPlayerInactivityTime())
         prepareDestroy();
+ 
+    if (QString::fromStdString(userInfo->privlevel()).toLower() == "none")
+        if ((server->getIdleClientTimeout() > 0) && (idleClientWarningSent == true))
+            if (timeRunning - lastActionReceived > server->getIdleClientTimeout())
+                prepareDestroy();
```

Comment:
I'd either wrap these conditional blocks with `{}` or put then inside one large `if` clause. The depth here with lack of scoping braces looks confusing.